internal/hcsoci/create.go

```diff
@@ -149,40 +151,40 @@ func CreateContainer(createOptions *CreateOptions) (_ *hcs.System, _ *Resources,
 	}
 
 	var hcsDocument interface{}
-	logrus.Debugf("hcsshim::CreateContainer allocating resources")
+	logrus.Debug("hcsshim::CreateContainer allocating resources")
 	if coi.Spec.Linux != nil {
 		if schemaversion.IsV10(coi.actualSchemaVersion) {
 			return nil, resources, errors.New("LCOW v1 not supported")
 		}
-		logrus.Debugf("hcsshim::CreateContainer allocateLinuxResources")
+		logrus.Debug("hcsshim::CreateContainer allocateLinuxResources")
 		err = allocateLinuxResources(coi, resources)
 		if err != nil {
-			logrus.Debugf("failed to allocateLinuxResources %s", err)
+			logrus.WithError(err).Debug("failed to allocateLinuxResources")
 			return nil, resources, err
 		}
 		hcsDocument, err = createLinuxContainerDocument(coi, resources.containerRootInUVM)
 		if err != nil {
-			logrus.Debugf("failed createHCSContainerDocument %s", err)
+			logrus.WithError(err).Debug("failed createHCSContainerDocument")
```

Comment:
I know this was just a transfer but it seems like `WithError(err)` should be `.Error` not `.Debug`

Reply:
I'm tempted to just remove this line because it doesn't provide any extra context; it's just another duplicate log message.

Reply:
I think I'll leave this for a separate change, though--there are a bunch of redundant such messages.

Reply:
I'm fine with that too